packages/starspot-json-api/src/serializer.ts

```diff
@@ -71,11 +71,11 @@ class Serializer {
     this.primaryResources.push(...result.resources);
   }
 
-  async serialize(model: any, included: Included): Promise<JSONAPI.DataDocument> {
+  async serialize(model: any, included: Included = new Included()): Promise<JSONAPI.DataDocument> {
```

Comment:
Does TypeScript automatically infer the type if you assign it? In other words, can this be condensed to just: `included = new Included()`?